incubator/virtualcluster/doc/demo.md

```diff
@@ -135,39 +273,59 @@ spec:
         command:
         - top
 EOF
-```
```

Comment:
Can you check the output is what you are looking for? it looks like it's all in a code block for the experiments section. Would it be easier to read for a user if the setups were broken by "normal" text between the code blocks?

Reply:
Makes sense, one single code block is really too much.
Let me update that accordingly.